packages/office-ui-fabric-react/src/components/Link/Link.test.tsx

```diff
@@ -29,4 +30,19 @@ describe('Link', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('renders Link with "as=div" a div element', () => {
+    const component = renderer.create(<Link renderAs='div' className='customClassName'>I'm a div</Link>);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders Link with "as=Route" a Route element', () => {
+    class Route extends React.Component {
+      public render() { return null; }
+    }
+    const component = mount(<Link renderAs={ Route } className='customClassName'>I'm a Route</Link>);
```

Comment:
using enzyme with snapshots produces ugly snapshots, compared to react-test-renderer. Do you guys have suggestions here on making this prettier (e.g. actually show the rendered divs and what their properties are, instead of Object) If now, can we just use test-renderer here?

Reply:
I don't have a good answer for that yet, i'll have a look to see what i can come up with, but we can surely use test-renderer :)